walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,144 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Arrays;
 
+// Meeting request: duration and attendes. 
+// Event: Time range, name of event, attendes of event. 
 public final class FindMeetingQuery {
+    
+  public ArrayList<TimeRange> gettingCalendar(
```

Comment:
Adding a comment to this method too, even if it's private, would probably be a good idea since it's not immediately clear what it's intended behavior is.